tests/integration/enums/python/__main__.py

```diff
@@ -40,4 +40,4 @@ def __init__(self, name: str, type: Input[RubberTreeVariety], farm: Optional[Inp
 
 export("myTreeType", tree.type)
 export("myTreeFarmChanged", tree.farm.apply(lambda x: x + "foo"))
-export("mySentence", Output.all(tree.type, tree.farm).apply(lambda args: f"My {args[0]} Rubber tree is from {args[1]}"))
+export("mySentence", Output.all(tree.type, tree.farm).apply(lambda args: f"My {args[0].value} Rubber tree is from {args[1]}"))
```

Comment:
@justinvp This may not be necessary if we use [StrEnum](https://docs.python.org/3/library/enum.html#enum.StrEnum) instead (new in 3.11) but I think your fix is better (in that it'll work with older versions of Python too).

@dixler Yeah, this is backwards compatible. This change in this test is from a change in behavior of Python's enum type, not any change in Pulumi. Given `FOO = "foo"` in an enum, `str(SomeEnum.FOO)` is now `"SomeEnum.FOO"` instead of `"foo"`, so this is explicitly requesting the value (`"foo"`).